src/azure-cli/azure/cli/command_modules/appconfig/_format.py

```diff
@@ -77,6 +81,17 @@ def _deleted_configstore_format_group(item):
     ])
 
 
+def _configstore_replica_format_group(item):
+    return OrderedDict([
```

Comment:
>OrderedDict [](http://example.com/codeflow?start=11&length=11)

It doesn't have the config store name?

Reply:
No, it doesn't have the name of the origin store.
Although it would be better to have the origin store name here, it may be unnecessary because the user needs to know the origin store name to view the replicas. So, they would be providing the name int the command.

Reply:
Most of those attributes are associated with the replica itself. But since we add the source group here, seems like we should also add resource level stuff. What about the json output format. It 1-to-1 maps all attributes?